apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -75,22 +75,23 @@ model DiscordUser {
 }
 
 model Guild {
-  id                             Int                  @id @default(autoincrement())
-  createdAt                      DateTime             @default(now())
-  updatedAt                      DateTime             @default(now()) @updatedAt
-  discord_id                     String?              @unique // legacy
+  id                             Int                          @id @default(autoincrement())
+  createdAt                      DateTime                     @default(now())
+  updatedAt                      DateTime                     @default(now()) @updatedAt
+  discord_id                     String?                      @unique // legacy
   name                           String? // legacy
   congrats_channel               String? // legacy
   logo                           String?
-  status                         GuildStatus          @default(VALIDATED)
+  status                         GuildStatus                  @default(VALIDATED)
   contributions                  GuildContribution[]
   users                          GuildUser[]
   twitter_accounts               TwitterAccount[]
   activity_type                  GuildActivityType[]
   contribution_reporting_channel String?
   guild_imports                  GuildImport[]
   verification_setting_id        Int?
-  verification_setting           VerificationSetting? @relation(fields: [verification_setting_id], references: [id])
+  verification_setting           VerificationSetting?         @relation(fields: [verification_setting_id], references: [id])
```

Comment:
This approach to declaring foreign key relations is consistent with the conventions used throughout the rest of the project.